cli/src/wallet_cli/mod.rs

```diff
@@ -145,12 +142,18 @@ pub enum WalletCommand {
     MintNft {
         /// Address to send the NFT to, e.g. rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3.
         address: Option<Bech32Address>,
+        /// Immutable metadata key, e.g. --immutable-metadata-key data.
+        #[arg(long, default_value = "data")]
```

Comment:
do we need group tags?

Reply:
No, the data group is because it can come from the CLI or a file, the key only comes from the CLI